classes/helper.php

```diff
@@ -494,11 +494,6 @@ public static function dismiss_notice($noticeid) {
             // Record dismiss action.
             self::create_new_acknowledge_record($noticeid, acknowledgement::ACTION_DISMISSED);
 
-            // Log user out.
```

Comment:
why do we change the logic here? In this case we logout a user before marking a notice as viewed. But after your change we mark it viewed and then log out.